build/illumos/build.sh

```diff
@@ -113,6 +113,10 @@ build_aarch64() {
         PKGPUBLISHER_REDIST=$PKGPUBLISHER
         DMAKE_MAX_JOBS=$MJOBS
         PYTHON3_VERSION=$PYTHONVER
+        BUILDPY3b=
+        PYTHON3b_VERSION=3.11
+        PYTHON3b_PKGVERS=-311
+        PYTHON3b_SUFFIX=
```

Comment:
keeping 3.11 around ?

Reply:
During the transition we need to deliver both 3.11 and 3.12 modules from illumos so that we can run `pkg` with either python version. Specifically there is a python libbe binding built from here.
Once we're completely on to 3.12 and 3.11 is removed, these lines will go too.